docs/oss/sql-blocks.md

```
@@ -0,0 +1,631 @@
+**SQL Block:**
+```yaml
+- type: sql
+  content: |
+    SELECT 
+      customer_id,
+      SUM(amount) as total_spent,
+      COUNT(*) as order_count
+    FROM orders
+    WHERE order_date >= '2024-01-01'
+    GROUP BY customer_id
+    ORDER BY total_spent DESC
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)